crates/ruff_linter/resources/test/fixtures/refurb/FURB156.py

```diff
@@ -0,0 +1,20 @@
+# Errors
+
+_ = "0123456789"
+_ = "01234567"
+_ = "0123456789abcdefABCDEF"
+_ = "abcdefghijklmnopqrstuvwxyz"
+_ = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+_ = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+_ = r"""!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~"""
+_ = " \t\n\r\v\f"
+
+_ = "" in "1234567890"
+_ = "" in "12345670"
+_ = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ!"#$%&\'()*+,-./:;<=>?@[\\]^_`{|}~ \t\n\r\x0b\x0c'
```

Comment:
Could you add a test using parentheses

```suggestion
_ = (
	'0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ!"#$%&\'()*+,-./:;<=>?@[\\]^_`{|}~ \t\n\r\x0b\x0c'
)
```

Reply:
https://github.com/astral-sh/ruff/pull/13530/commits/066e7006e4b5fad341fa8c84a85ec1f5cf5d3602
I added the test and "took a look" on `parenthesized_range`, but its usage was really not intuitive for me in that special case. Could I somehow simplify "getting the parent" here?

Reply:
I think it's fine to preserve the parentheses in this case. That also guarantees that we preserve (most) comments. I'm sorry that I set you off on the wrong trail.